apps/docs/src/examples/test/Box.svelte

```diff
@@ -1,32 +1,32 @@
 <script lang="ts">
-  import { forwardEventHandlers, T } from '@threlte/core'
+  import { T } from '@threlte/core'
   import { useCursor } from '@threlte/extras'
   import { spring } from 'svelte/motion'
 
-  let color = 'white'
+  let { ...props } = $props()
+
+  let color = $state('white')
 
   const scale = spring(1)
-  const component = forwardEventHandlers()
 
   const { onPointerEnter, onPointerLeave } = useCursor()
 </script>
 
 <T.Group
   scale={$scale}
-  {...$$restProps}
-  bind:this={$component}
+  {...props}
```

Comment:
nice. that's a good improvement on ergonomics imo

Reply:
Agreed!